deps/wazuh_testing/wazuh_testing/qa_ctl/provisioning/wazuh_deployment/wazuh_s3_package.py

```diff
@@ -1,11 +1,12 @@
+from logging import setLoggerClass
```

Comment:
What is this class?

Reply:
This requested change can be reviewed in the commit : [5c3a34d2757275d66bb9951667083965a0452fab](https://github.com/wazuh/wazuh-qa/commit/5c3a34d2757275d66bb9951667083965a0452fab)